Library/Homebrew/keg_relocate.rb

```diff
@@ -186,6 +238,26 @@ def each_unique_file_matching(string)
     end
   end
 
+  def each_unique_binary_file
+    grep_bin, grep_args = egrep_args
+
+    # An extra \ is needed for the null character when calling grep
+    Utils.popen_read(grep_bin, grep_args, "\\x00", to_s) do |io|
```

Comment:
```suggestion
    Utils.popen_read(grep_bin, grep_args, "\\#{BINARY_NULL_CHARACTER}", to_s) do |io|
```

Does something like this work?